app/services/payment_provider_customers/adyen_service.rb

```diff
@@ -20,6 +20,11 @@ def create
 
       result.checkout_url = checkout_url_result.checkout_url
       result
+    rescue Adyen::AuthenticationError
```

Comment:
arguably this should be `call` instead of `create`, You probably thought the same and decided it was too much work, just wanted to have that confirmed :D

Reply:
@nudded exactly :D
I started to refact it and... it was A LOT of changes... haha.
I'm working on a proposal to tackle all of those refactors once for all...